questions/views.py

```diff
@@ -136,6 +138,56 @@ def bulk_create_forecasts_api_view(request):
     return Response({}, status=status.HTTP_201_CREATED)
 
 
+@api_view(["POST"])
+@transaction.non_atomic_requests
+def bulk_withdraw_forecasts_api_view(request):
+    now = timezone.now()
+    serializer = ForecastWithdrawSerializer(data=request.data, many=True)
+    serializer.is_valid()
+
+    validated_data = serializer.validated_data
+
+    if serializer.errors:
+        raise ValidationError({"errors": serializer.errors})
+
+    if not serializer.validated_data:
+        raise ValidationError("At least one forecast must be withdawn")
```

Comment:
Replace all of these checks with just `serializer.is_valid(raise_exception=True)`. This will raise ValidationError in case of wrong data and will be automatically converted to the appropriate JSON response by Django